Whatsapp_Automated_Messages/__main__.py

```diff
@@ -0,0 +1,103 @@
+import pyautogui as pt 
+import time
+import pyperclip
+import random 
+
+time.sleep(4)
+
+position1 = pt.locateOnScreen("Whatsapp_Automated_Messsages/smilie_paperclip.png", confidence=.6)
+x= position1[0]
+y= position1[1]
+
+
+def get_message():             # it will help us to fetch the messages
+    global x, y
+
+    position = pt.locateOnScreen("Whatsapp_Automated_Messsages/smilie_paperclip.png", confidence=.6)
```

Comment:
you have defined position1 before, isn't that same?

Reply:
No actually not

Reply:
```python
position1 = pt.locateOnScreen("./Whatsapp_Automated_Messsages/smilie_paperclip.png", confidence=.6)
position = pt.locateOnScreen("./Whatsapp_Automated_Messsages/smilie_paperclip.png", confidence=.6)
```
What's the difference?

Reply:
I don't know how to put it into words
When I was writing the function I felt its necessary to introduce a new variable to improve readability, improve how I interact with my own code, and more importantly the two variables here have values that do very different functions and it was needed to introduce two variables and not one.

Reply:
You can make position variable global which can be used anywhere in the function. Second, duplicate variables doesn't make code readable